site/js/main.js

```diff
@@ -139,6 +144,19 @@ var drums = new Pz.Sound({
 	drums.addEffect(convolver); 
 });
 
+var tremoloDrums = new Pz.Sound({ 
```

Comment:
I think a guitar sound will make the effect stand out more, but let's leave it like this for the time being. I will try to record one after we make a merge.